tools/main.cpp

```diff
@@ -66,5 +65,9 @@ int main(int argc, char** argv) {
 
    CLI11_PARSE(app, argc, argv);
 
-   return runner.exec();
+   try {
+      return runner.exec();
+   } catch(...) {
+      return -1;

```

Comment:
it would be nice to have more verbose exceptions handling unless runner is not supposed to print all the details